endpoints/openrtb2/amp_auction_test.go

```diff
@@ -120,7 +120,11 @@ func TestGoodAmpRequests(t *testing.T) {
 				}
 			}
 			if test.ExpectedValidatedBidReq != nil {
-				assert.Equal(t, test.ExpectedValidatedBidReq, ex.actualValidatedBidReq, "Not the expected validated request. Test file: %s", filename)
+				// compare as json to ignore whitespace and ext field ordering
+				actualJson, err := json.Marshal(ex.actualValidatedBidReq)
+				if assert.NoError(t, err, "Error converting actual bid request to json. Test file: %s", filename) {
+					assert.JSONEq(t, string(test.ExpectedValidatedBidReq), string(actualJson), "Not the expected validated request. Test file: %s", filename)
+				}
```

Comment:
Tests failed when I ran a standard json formatter and it broke the ext comparisons. Moved to json compare to ignore whitespace and field ordering.